hooks/tk-multi-shotgunpanel/basic/scene_actions.py

```diff
@@ -158,33 +158,42 @@ def execute_action(self, name, params, sg_data):
                                 publish fields.
         """
         app = self.parent
-        app.log_debug(
+        app.logger.debug(
             "Execute action called for action %s. "
             "Parameters: %s. Publish Data: %s" % (name, params, sg_data)
         )
 
-        # resolve path
-        # toolkit uses utf-8 encoded strings internally and the After Effects API expects unicode
-        # so convert the path to ensure filenames containing complex characters are supported
-        path = self.get_publish_path(sg_data).decode("utf-8")
-
-        if self.parent.engine.is_adobe_sequence(path):
-            frame_range = self.parent.engine.find_sequence_range(path)
-            if frame_range:
-                glob_path = re.sub(
-                    r"[\[]?([#@]+|%0\d+d)[\]]?", "*{}".format(frame_range[0]), path
-                )
-                for each_path in sorted(glob.glob(glob_path)):
-                    path = each_path
-                    break
-
-        if not os.path.exists(path):
-            raise Exception("File not found on disk - '%s'" % path)
-
-        if name == _ADD_TO_COMP:
-            self._add_to_comp(path)
-        if name == _ADD_TO_PROJECT:
-            self.parent.engine.import_filepath(path)
+        # This hook only implements the add to comp and project logic, fall back to the
+        # base class for anything else.
+        if name in [_ADD_TO_COMP, _ADD_TO_PROJECT]:
```

Comment:
Not sure I understand here. Are you saying that if someone derived from this hook and it would break?

Reply:
Oh, it never called the base class, I see now, so you didn't get the default actions on publishes, correct?

Reply:
Yeah that's correct, I mentioned it in the description of the PR, but it would probably have been better to drop the comment in on the code like this.